planning/behavior_path_start_planner_module/src/manager.cpp

```diff
@@ -335,14 +337,333 @@ void StartPlannerModuleManager::updateModuleParams(
+    updateParam<double>(
+      parameters, safety_check_ns + "backward_path_length",
+      p->safety_check_params.backward_path_length);
+    updateParam<double>(
+      parameters, safety_check_ns + "forward_path_length",
+      p->safety_check_params.forward_path_length);
+    updateParam<bool>(
+      parameters, safety_check_ns + "publish_debug_marker",
+      p->safety_check_params.publish_debug_marker);
+  }
+  const std::string rss_ns = safety_check_ns + "rss_params.";
+  {
+    updateParam<double>(
+      parameters, rss_ns + "rear_vehicle_reaction_time",
+      p->safety_check_params.rss_params.rear_vehicle_reaction_time);
+    updateParam<double>(
+      parameters, rss_ns + "rear_vehicle_safety_time_margin",
+      p->safety_check_params.rss_params.rear_vehicle_safety_time_margin);
+    updateParam<double>(
+      parameters, rss_ns + "lateral_distance_max_threshold",
+      p->safety_check_params.rss_params.lateral_distance_max_threshold);
+    updateParam<double>(
+      parameters, rss_ns + "longitudinal_distance_min_threshold",
+      p->safety_check_params.rss_params.longitudinal_distance_min_threshold);
+    updateParam<double>(
+      parameters, rss_ns + "longitudinal_velocity_delta_time",
+      p->safety_check_params.rss_params.longitudinal_velocity_delta_time);
+  }
+  std::string surround_moving_obstacle_check_ns = ns + "surround_moving_obstacle_check.";
+  {
+    updateParam<double>(
+      parameters, surround_moving_obstacle_check_ns + "search_radius", p->search_radius);
+    updateParam<double>(
+      parameters, surround_moving_obstacle_check_ns + "th_moving_obstacle_velocity",
+      p->th_moving_obstacle_velocity);
+
+    // ObjectTypesToCheck
+    std::string obj_types_ns = surround_moving_obstacle_check_ns + "object_types_to_check.";
+    {
+      updateParam<bool>(
+        parameters, obj_types_ns + "check_car",
+        p->surround_moving_obstacles_type_to_check.check_car);
+      updateParam<bool>(
+        parameters, obj_types_ns + "check_truck",
+        p->surround_moving_obstacles_type_to_check.check_truck);
+      updateParam<bool>(
+        parameters, obj_types_ns + "check_bus",
+        p->surround_moving_obstacles_type_to_check.check_bus);
+      updateParam<bool>(
+        parameters, obj_types_ns + "check_trailer",
+        p->surround_moving_obstacles_type_to_check.check_trailer);
+      updateParam<bool>(
+        parameters, obj_types_ns + "check_unknown",
+        p->surround_moving_obstacles_type_to_check.check_unknown);
+      updateParam<bool>(
+        parameters, obj_types_ns + "check_bicycle",
+        p->surround_moving_obstacles_type_to_check.check_bicycle);
+      updateParam<bool>(
+        parameters, obj_types_ns + "check_motorcycle",
+        p->surround_moving_obstacles_type_to_check.check_motorcycle);
+      updateParam<bool>(
+        parameters, obj_types_ns + "check_pedestrian",
+        p->surround_moving_obstacles_type_to_check.check_pedestrian);
     }
-  });
+  };
```

Comment:
```suggestion
  }
```
As CI said

> planning/behavior_path_start_planner_module/src/manager.cpp:666:  You don't need a ; after a }  [readability/braces] [4]
Done processing planning/behavior_path_start_planner_module/src/manager.cpp

https://results.pre-commit.ci/run/github/438106587/1709632306.q_aUiOkERIGEU-e5Zyj2mw